### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by Jiwei99**
Contributes to #171

### CODE REVIEW

File: docs/team/jiwei99.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  * Managed issues and milestones for `v1.3` and `v1.3b`.
+  
+
+* **Enhancements to existing feature**: Improved `Help` command usability.
+  * Added functionality to allow `Help` to take in additional parameters so that users can easily obtain support for usage of commands. For example, users can now use `help` to obtain a list of available commands, and `help COMMAND` to get help for a specific command.
+  * Refactored help messages for all commands to improve quality of help messages.
+
+
+* **Enhancements to existing feature**: Added unit test for functions.
+  * Added unit tests to for new and existing features to increase test coverage.
```
Comment (yucheng11122017) [inline docs/team/jiwei99.md:45]: Maybe you can specific which features did you do that for?

File: docs/team/jiwei99.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ 
+ 
+* **Documentation**:
+  * User Guide:
+      * Updated and refactored User Guide to align with features of CONNECTIONS ([\#20](https://github.com/AY2122S1-CS2103-F09-4/tp/pull/20))
+      * Added documentation for BIRTHDAY parameter ([\#20](https://github.com/AY2122S1-CS2103-F09-4/tp/pull/20))
```
Comment (yucheng11122017) [inline docs/team/jiwei99.md:51]: The spacing not consistent with below

**Review by yucheng11122017 — APPROVED** [review]
LGTM except for minor changes :)

**Review by Justinhoejj — APPROVED** [review]
LGTM!

**Pull request merged**